SU2_CFD/src/output/COutput.cpp

```diff
@@ -129,12 +128,29 @@ COutput::COutput(CConfig *config, unsigned short nDim, bool fem_output): femOutp
   cauchyValue = 0.0;
   convergence = false;
 
+  /*--- Initialize time convergence monitoring structure ---*/
+
+  nWndCauchy_Elems = config->GetWnd_Cauchy_Elems();
+  wndCauchyEps     = config->GetWnd_Cauchy_Eps();
+
+  for (unsigned short iField = 0; iField < config->GetnWndConv_Field(); iField++){
+    wndConvFields.emplace_back(config->GetWndConv_Field(iField));
+  }
```

Comment:
General note (not very important in this case) whenever you know how many things you are inserting into the vector you should `reserve` before.

Reply:
Okay, I added it nevertheless. Thanks for the feedback!